### PR TITLE
Collaboration on portfolio form-section

### PR DESCRIPTION
**Pull request by mengiefen**
-The form section has been made.
-Inside this section, input elements with form validation are implemented.
-Additionally, a footer section is added, to navigate to social media.

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -223,5 +223,93 @@ <h1 class="aboutme-heading">About Me</h1>
...
+            ><img
+              src="/assets/images/icon-twitter-dark.svg"
+              alt="Twitter icon"
+              class="twitter-icon"
+          /></a>
+          <a href="#"
+            ><img
+              src="/assets/images/icon-medium-dark.svg"
+              alt="Medium icon"
+              class="medium-icon"
```
Comment (micheaol) [inline home.html:308]: Please ensure you stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding).

## Your design:
your-design

## Figma design:
figma

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @mengiefen , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT 🥇
- Your project is professional 
- Your repo is professional 
- You set up linter files and no errors. 

## Required Changes ♻️
- _Please check the comments under the review._
- Please add your project screenshot to the README file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: home.html (hunk 1 above)
Comment (mengiefen) [reply home.html:308]: Hi @micheaol 

From your suggestion, I can see that the "textarea" background is different from the Figma reference design file. But I am unable to figure out the other details that require change. Would you please point me to which portion I need to apply the change?

Comment (micheaol) [reply home.html:308]: @mengiefen 
Please also ensure that the social icons are visible and the contents are properly aligned just like Figma's design.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @mengiefen 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**